enterprise/frontend/src/embedding-sdk/.eslintrc

```diff
@@ -4,6 +4,22 @@
       "error",
       {
         "paths": [
+          {
+            "name": "@mantine/core",
+            "message": "Please import from `metabase/ui` instead."
+          },
```

Comment:
can you add this rule to `enterprise/frontend/.eslintrc` as well for consistency? I just tested it real quick and I can still import `@mantine/core` outside the sdk folder (`/enterprise/frontend/src` for example)

Reply:
Done, I copied some other `paths` over in a few places